encryptonize_test.go

```diff
@@ -1320,3 +1321,49 @@ func TestSharingObjectPart4(t *testing.T) {
 		}
 	}
 }
+
+func TestAddToIndex(t *testing.T) {
+	enc := newTestEncryptonize(t)
+
+	index := NewIndex()
```

Comment:
`NewIndex` should not be exported, it should be exposed through the `Encryptonize` struct. Come to think of it, `SealedID` should also be private.

Reply:
Fixed